src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.cs

```diff
@@ -738,7 +738,7 @@ private static void ValidateCspFlags(CspProviderFlags flags)
                 (keyType == CspAlgorithmType.Dss && dwAlgId != CALG_DSS_SIGN))
             {
                 hKey.Dispose();
-                throw new CryptographicException(SR.Format(SR.Cryptography_CSP_WrongKeySpec, Convert.ToString(keyType)));
+                throw new CryptographicException(SR.Cryptography_CSP_WrongKeySpec);
```

Comment:
Same here also, is it worth formatting `Convert.ToString(keyType)` into the message.